### ISSUE TITLE
Error updating light effects

### ISSUE THREAD

**Issue by tcutchi** [Aurora-RGB/Aurora#95]
### Expected Behaviour
The keyboard lights up per profile colors

### Actual Behaviour
Aurora throws an exception "Error while updating light effects: Parameter is not valid" 
on startup, logs attached.

[2022-10-26 16.55.57.log](https://github.com/Aurora-RGB/Aurora/files/9874013/2022-10-26.16.55.57.log)

I am now using an Intel 13th Gen processor (13700K) on an Asus ROG_STRIX_Z690-A_GAMING_WIFI_D4 motherboard. Log suggest an issue with Aurora being unable to find CPU Temp and Power sensors.

### Reproduction steps
Unknown

### Frequency
Every time?

### Aurora Version:
V76

### Previously an Issue?
Don't know

**Comment by Aytackydln** [comment]
Hmm this is completely new... The library Aurora uses has an open issue already. I will push the fix as soon as it is updated. Meantime, I will try to make faulty sensor readings at least empty value to make Aurora usable with them.

Does this happen with default profile?

**Comment by tcutchi** [comment]
Hello, after backing up my profiles and reinstalling from scratch, I can say it did not happen with the built-in default profile. Reloading my saved profile (which does not contain CPU power or temp, but DOES contain CPU, RAM, and GPU usage) caused the crash again. 

Oddly enough, this was not happening yesterday, though no changes have been made to either the profile or the system.

Today's events:

1st launch - Crash
After reinstalling, with default profile - No crash, but crash again on second launch
[2022-10-26 17.36.58.log](https://github.com/Aurora-RGB/Aurora/files/9874204/2022-10-26.17.36.58.log)

Added back my saved profile - Crashed same as before, but exiting and launching Aurora again, custom profile is working as intended.
[2022-10-26 17.49.26.log](https://github.com/Aurora-RGB/Aurora/files/9874285/2022-10-26.17.49.26.log)

It seems reinstalling fixed it somehow despite this already being a new install (for V76 as of a couple hours ago)

**Comment by Aytackydln** [comment]
This problem seems to be solved in reading library's last version. I will include it in next update

**Comment by Aytackydln** [comment]
https://github.com/Aurora-RGB/Aurora/releases/tag/v81

can you confirm this is solved?

**Comment by tcutchi** [comment]
Hi, I'm not actually sure. I did install v81, and the update window now
pops up much less frequently. However I can no longer see the main window
at all, and the lighting is also not working (I think it's because Aurora
starts up faster than Corsair iCue, and so it's not detecting it and
starting the plugin, but I also can't get to the settings).
Manually starting Aurora by exiting and relaunching makes the keyboard
lighting work, but there's still no window. Hovering over the icon, I just
see this:
[image: image.png]
Clicking on it does not show a visible window.

I just downloaded and installed v82 as well, same issue.

I will try and fully uninstall and reinstall. Can you point me to where the
profile files are located? I have some backups that I know are working but
they are slightly dated.

Thanks!

**Comment by Aytackydln** [comment]
profile data is at %appdata%/Aurora 
Can you try maximizing by hovering the taskbar icon then right clicking the popup?

**Comment by Aytackydln** [comment]
also you can increase aurora startup delay

**Comment by tcutchi** [comment]
Reinstalling worked. (V82)
No update popup, and I was able to set the startup delay as well. I tested
with a couple of shutdowns and restarts and did not see any issues.

Thank you!

If you're able I'd like to request a couple of peripherals to be added -
Logitech G604 mouse (1 RGB LED) and G633/G933/G635/G935 headsets (2
zones, logo and side stripe). The headsets are all variants of the same.
x35 is a newer variant of x35, and 6xx are wired vs 9xx being wireless.
I currently use a 633, and if I set the "mouse" to either G900 or G502,
both zones on the headset will light up with the "Logo" assignment. Nothing
on the mouse though.
